test/uk/gov/hmrc/gvmshaulierstatus/controllers/HaulierStatusControllerSpec.scala

```diff
@@ -64,6 +64,7 @@ class HaulierStatusControllerSpec extends BaseControllerSpec {
       status(result) shouldBe CREATED
 
       verify(mockHaulierStatusService).create(mEq(correlationId))
+      verify(mockStubBehaviour).stubAuth(mEq(Some(expectedPermissionPredicate)), mEq(Retrieval.EmptyRetrieval))
```

Comment:
i think we can add this inside Setup class just once  rather than adding to all tests ?

Reply:
there's one case where we want to make sure it's never called, in the case where the token is missing (I think it was)

Reply:
might be worth separating it out and making it composable, but I don't think it's a biggie